apps/passport-client/components/screens/FrogScreens/ScoreTab.tsx

```diff
@@ -25,6 +42,17 @@ export function ScoreTab({ score }: { score?: FrogCryptoScore }) {
 
   return (
     <Container>
+      {
+        // only show share button if user has a telegram username
+        score.has_telegram_username && (
```

Comment:
```suggestion
        score?.has_telegram_username && (
```

plz be careful, here and elsewhere. unfortunately we don't have `strict` mode enabled in the zupass client `tsconfig`.

i've been seeing more 'cannot read property X of undefined' in rollbar, and i suspect it's coming from frog crypto. eg https://app.rollbar.com/a/pcd-team/fix/item/zupass/1353

could you do a pass through your code to make sure to consider `undefined` cases?

Reply:
we have a guard against score not defined on L39. but I'll make a pass on the undefined issue.

also TIL I have seen TS not inferring null-ness correctly and was very confused